src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -4015,6 +4015,16 @@ theorem toNat_rotateLeft {x : BitVec w} {r : Nat} :
     (x.rotateLeft r).toNat = (x.toNat <<< (r % w)) % (2^w) ||| x.toNat >>> (w - r % w) := by
   simp only [rotateLeft_def, toNat_shiftLeft, toNat_ushiftRight, toNat_or]
 
+theorem toInt_rotateLeft {x : BitVec w} {r : Nat} :
+  (x.rotateLeft r).toInt =
+    ((x <<< (r % w)).toNat ||| (x >>> (w - r % w)).toNat : Int).bmod (2 ^ w) := by
+  simp only [rotateLeft_def, toInt_shiftLeft, toInt_ushiftRight, toInt_or]
```

Comment:
Does this need to be simp_only, or can it be just `simp`?

Reply:
can be `simp`, just replaced!